packages/vscode-extension-serverless-workflow-editor/src/extension/serviceCatalog/serviceRegistry/ServiceRegistriesStore.ts

```diff
@@ -140,4 +153,22 @@ export class ServiceRegistriesStore {
   private notifyRefresh() {
     this.subscriptions.forEach((subscription) => subscription(this.storedServices));
   }
+
+  private checkSettingsAuthProviders(registriesSettings: SwfServiceRegistriesSettings) {
+    if (this.authProviderFactory.isRHAuthEnabled) {
+      return;
+    }
+
+    const needsRHAccount = registriesSettings?.registries?.find(
```

Comment:
```suggestion
    const needsRedHatAccount = registriesSettings?.registries?.find(
```

or 

```suggestion
    const needsRhAccount = registriesSettings?.registries?.find(
```

We do not use upper-case acronyms, as they can be very confusing when put together. (e.g. `RHTHTTPCDIService` This is a stupid name but you get the point 😝 )

This is a convention of our codebase, but there's no definitive answer, of course. See this discussion: https://stackoverflow.com/questions/8277355/naming-convention-for-upper-case-abbreviations